src/ucp/core/ucp_mm.c

```diff
@@ -1568,6 +1568,9 @@ ucs_status_t ucp_mem_rcache_init(ucp_context_h context,
         }
     }
 
+    context->config.ext.rcache_overhead = ucs_linear_func_make(
+            ucs_time_units_to_sec(rcache_config->overhead, 50.0e-9), 0);
```

Comment:
seems the prev default was 180e-9, why changed?

Reply:
As I understand it is changed to 50 since before that patch that value was mostly used for UCT level rcache, but now it is used for UCP level rcache, and the latest measurements on Selene shown that it is 50 nanoseconds which is exposed inside `UCP_RCACHE_LOOKUP_FUNC`

Reply:
shall we reuse (or delete) UCP_RCACHE_LOOKUP_FUNC?

Reply:
Updated macro.